doc/releases/changelog-dev.md

```diff
@@ -131,6 +131,23 @@
 * Internal use of in-place inversion is eliminated in preparation for its deprecation.
   [(#2965)](https://github.com/PennyLaneAI/pennylane/pull/2965)
 
+* `qml.simplify` can now be used to simplify quantum functions, tapes and QNode objects.
+  [(#2978)](https://github.com/PennyLaneAI/pennylane/pull/2978)
+
+  ```python
+    dev = qml.device("default.qubit", wires=2)
+    @qml.simplify
+    @qml.qnode(dev)
+    def circuit():
+      qml.adjoint(qml.prod(qml.RX(1, 0) ** 1, qml.RY(1, 0), qml.RZ(1, 0)))
+      return qml.probs(wires=0)
+  ```
+
+  ```pycon
+  >>> list(circuit.tape)
```

Comment:
```suggestion
  >>> circuit()
  >>> list(circuit.tape)
```

Once we improve how we draw arithmetic ops, we should use that here instead. But given that doesn't provide the right information yet, we should just make the tape is constructed first before we print out its contents.